bodhi/tests/server/scripts/test_approve_testing.py

```diff
@@ -469,19 +523,20 @@ def test_subsequent_comments_after_initial_push_comment(self):
         self.assertEqual(cmnts[2].text, config.get('testing_approval_msg'))
 
     @patch('sys.exit')
-    @patch('sys.stdout', new_callable=StringIO)
-    def test_usage(self, stdout, exit):
+    @patch('sys.stderr', new_callable=StringIO)
+    def test_usage(self, stderr, exit):
         """
         Assert that the usage message is returned to the user if not exactly 2 arguments are given
         """
         with patch('bodhi.server.scripts.approve_testing.initialize_db'):
             with patch('bodhi.server.scripts.approve_testing.get_appsettings', return_value=''):
-                approve_testing.main(['nosetests', 'some_config.ini', 'testnoses'])
+                approve_testing.main(['some_config.ini', 'testnoses'])
 
         self.assertEqual(
-            stdout.getvalue(),
-            'usage: nosetests <config_uri>\n(example: "nosetests development.ini")\n')
-        exit.assert_called_once_with(1)
+            stderr.getvalue(),
+            'usage: py.test-3 [-h] [--debug] config_uri\n'
+            'py.test-3: error: unrecognized arguments: testnoses\n')
```

Comment:
This fails in local tests because I call pytest as `pytest` instead of `py.test-3`. Either take the actual executable name into account here, or perhaps use a regex match?

Reply:
Already have the fix for this :)

Reply:
👍